tests/torch/test_loggers.py

```diff
@@ -0,0 +1,111 @@
+# --------------------------------------------------------------------------------------
+# Part of the interTwin Project: https://www.intertwin.eu/
+#
+# Created by: Anna Elisa Lappe
+#
+# Credit:
+# - Anna Lappe <anna.elisa.lappe@cern.ch> - CERN
+# -------------------------------------------------------------------------------------
+
+from argparse import Namespace
+from unittest.mock import MagicMock
+
+import pytest
+from lightning.pytorch.callbacks.model_checkpoint import ModelCheckpoint
+
+from itwinai.loggers import Logger
+from itwinai.torch.loggers import PyTorchLightningLogger
+
+
+@pytest.fixture
+def lightning_loggers():
+    """Setup a generic PyTorchLightningLogger with mock loggers for testing."""
+    itwinai_logger_mock = MagicMock()
+    lightning_logger = PyTorchLightningLogger(itwinai_logger=itwinai_logger_mock)
+
+    return itwinai_logger_mock, lightning_logger
+
+
+def test_lightning_experiment_property(lightning_loggers):
```

Comment:
I have some trouble understanding exactly what this is testing. The function that is being tested has at most three lines of actions, two under the if statement and one return statement. This test checks that the output type is correct and that one of the functions inside the if statements is being called. 

In other words, this test really just seems to be replicating the actual implementation of the code, which doesn't really offer much value beyond the original code itself. If you wanted to take this one step further, you could check that "self._initialized == True", at which point the test would be equivalent to the actual implementation itself. This means that if you ever wanna change the implementation but leave the signature the same, you would have to rewrite the tests even if the function does what it's supposed to.

Reply:
Hm so you mean you would suggest removing this test? Maybe I can add a shorter version of this to the function to test the .log call, as there the experiment is also initialised

Reply:
To be honest, I am not entirely sure. It might be useful, but when comparing it to the actual function implementation I feel like they are so similar that I am not sure what the test adds. That being said, I am unfamiliar with Lightning so I don't think I'd be able to fully assess the value of the test. It's more meant as "food for thought" and a way for me to grasp more of the domain than anything else, really :D